PatchXR/documentation
# Update docs with relatedBlocks (NewPatch/#1630)

Regenerated all documentation with the `relatedBlocks` field fixed.

The .json files were fixed in the NewPatch repo: https://github.com/PatchXR/NewPatch/pull/1631

To test:
- Checkout this branch on your local machine
- Open _builds/html/index.html_ in your browser and check everything is OK.
